org.kde.kdenlive.json

```diff
@@ -852,7 +853,7 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "http://git.sesse.net/movit",
+                    "url": "https://git.sesse.net/movit",
```

Comment:
Movit support has been temporarily removed in Kdenlive for the time being.

Reply:
Yes, we could remove it from the dependencies with 24.02